ichnaea/data/tests/test_export.py

```diff
@@ -154,10 +168,10 @@ def mock_send(self, data, num=num):
 
 class TestGeosubmit(BaseExportTest):
     def test_upload(self, celery, session, metricsmock):
-        ApiKeyFactory(valid_key="e5444-794")
+        ApiKeyFactory(valid_key="e5444-7946")
```

Comment:
I changed this to match the ``api_key`` in the report. Its not clear how this worked before.